incubator/tensorflow-inception/values.yaml

```diff
@@ -0,0 +1,15 @@
+# Default values for tensorflow-inception.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+ServiceType: LoadBalancer
+ServicePort: 9090
+ContainerPort: 9090
+Component: "tensorflow-inception"
+Replicas: 1
+Image: "quay.io/lachie83/inception_serving"
+ImageTag: "latest"
```

Comment:
Personally not a fan of latest ;) will defer to who merges the PR. I have had production failures because latest was not tested on k8s.